buddybuild_prebuild.sh

```diff
@@ -8,3 +8,15 @@ brew install swiftlint
 else
 echo "swiftlint already installed"
 fi
+
+if [ "$BUDDYBUILD_SCHEME" = "uispecs" ]; then
+
+# Check if python is installed
+python3 --version
+
+cd scripts/
+# Install cryptography
+pip3 install PyFxA syncclient cryptography
+# Run script to upload new login
+python3 upload_fake_passwordsBB.py 1
```

Comment:
If we run this on each buddybuild build won't the test account just get more and more credentials?

Reply:
In theory, and what I have seen doing some tests, no, I have added the condition: `f [ "$BUDDYBUILD_SCHEME" = "uispecs" ]`so that credential is only added when the xcuitest schema is launched to run the tests. I only see that happening when we run the tests, and so the credential will be removed. 
If you think there is a better approach, please let me know! I have been thinking about this and came with this solution but there may be a better one... 🤔